Formula/root.rb

```diff
@@ -30,6 +29,8 @@ class Root < Formula
     end
   end
 
+  users_from_macos "libxml2"
```

Comment:
This needs to be after the `depends_on` lines, not before.

Reply:
Oops, should have run `brew audit root --online --strict` but forgot.

Reply:
Okay, repushed, ran the audit this time too.